cell_based/src/cell/srn/CellSrnModel.hpp

```diff
@@ -109,75 +112,82 @@ class CellSrnModel : public AbstractSrnModel
     ~CellSrnModel();
 
     /**
-     * Initialize constituent SRN models
+     * Initialize constituent SRN models.
      */
-    virtual void Initialise() override;
+    virtual void Initialise();
 
     /**
-     * Calls SRN model specific behaviour at the time of cell division
+     * Calls SRN model specific behaviour at the time of cell division.
      * All constituent SRNs models (edge and/or interior, if there are any) call their
-     * implementation of this method
+     * implementation of this method.
      */
-    virtual void ResetForDivision() override;
```

Comment:
Do we have a policy on these new `override` things? From what I have read a good idea to include, but more for human and preventing human error than for the compiler?

Reply:
One for @fcooper8472 to comment on? I removed them here for consistency with the rest of the codebase, but if we want to include them everywhere that's fine (and probably should be part of another issue related to more extensive use of modern C++, like `auto`).

Reply:
@fcooper8472 says we should have override in all of the appropriate places, but we'll do this as part of another issue looking at modernising the C++, use of keywords etc.